.github/workflows/fast_testing.yml

```diff
@@ -10,6 +10,48 @@ on:
       - '*'
 
 jobs:
+  version_check:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - '1.10'
+          - '2.8'
+          - '2.10'
```

Comment:
I think you can install only latest here.

Reply:
Done with 2.10